src/components/Home/Home.module.css

```diff
@@ -0,0 +1,13 @@
+.container {
```

Comment:
Good idea. but dont start leaking out additional stuff onto this branch. This branch should be all about the nav. not home page styling and building. Keep things small and separate going forward.